handlers/handlers_test.go

```diff
@@ -240,4 +244,342 @@ func TestHandler(t *testing.T) {
+							codelist.EditionsList{
+								Edition: "2018",
+								Label:   "Local authority districts",
+								Links:   codelist.EditionsListLink{},
+							},
+						},
+						Count:      1,
+						Offset:     0,
+						Limit:      1,
+						TotalCount: 1,
+					}, nil
+				},
+				GetCodeByIDFunc: func(codeListID string, edition string, codeID string) (codelist.CodeResult, error) {
+					return codelist.CodeResult{
+						ID:    "E07000223",
+						Label: "Adur",
+					}, nil
+				},
+				GetDatasetsByCodeFunc: func(codeListID string, edition string, codeID string) (codelist.DatasetsResult, error) {
+					return codelist.DatasetsResult{
+						Datasets: []codelist.Dataset{
+							codelist.Dataset{
+								Links:          codelist.DatasetLinks{},
+								DimensionLabal: "Adur",
+								Editions: []codelist.DatasetEdition{
+									codelist.DatasetEdition{
+										Links: codelist.DatasetEditionLink{
+											Self:            codelist.Link{},
+											DatasetDimenion: codelist.Link{},
+											LatestVersion: codelist.Link{
+												ID:   "1",
+												Href: "http://localhost:22000/datasets/mid-year-pop-est/editions/time-series/versions/1",
+											},
+										},
+									},
+								},
+							},
+						},
+						Count: 1,
+					}, nil
+				},
+			}
+			mockDatasetClient := &DatasetClientMock{
+				GetFunc: func(ctx context.Context, id string) (dataset.Model, error) {
+					return dataset.Model{
+						Description: "Test dataset description",
+						Title:       "Test dataset title",
+					}, nil
+				},
+			}
+
+			router.Path("/geography/{codeListID}/{codeID}").HandlerFunc(AreaPageRender(mockRenderClient, mockCodeListClient, mockDatasetClient))
+			router.ServeHTTP(w, req)
+			renderCall := mockRenderClient.DoCalls()[0]
+
+			var payload area.Page
+			err := json.Unmarshal(renderCall.In2, &payload)
+			if err != nil {
+				t.Errorf("Failed to unmarshal payload sent to renderer to relevant frontend model: %s", err)
+				return
+			}
+
+			So(payload.Metadata.Title, ShouldEqual, "Adur")
+			So(payload.Breadcrumb, ShouldResemble, []model.TaxonomyNode{
+				model.TaxonomyNode{
+					Title: "Home",
+					URI:   "https://www.ons.gov.uk",
+				},
+				model.TaxonomyNode{
+					Title: "Geography",
+					URI:   "/geography",
+				},
+				model.TaxonomyNode{
+					Title: "Local authority districts",
+					URI:   "/geography/local-authority",
+				},
+				model.TaxonomyNode{
+					Title: "Adur",
+					URI:   "/geography/local-authority/E07000223",
+				},
+			})
+			So(len(payload.Data.Datasets), ShouldEqual, 1)
+			So(payload.Data.Datasets, ShouldResemble, []area.Dataset{
+				area.Dataset{
+					ID:          "",
+					Label:       "Test dataset title",
+					Description: "Test dataset description",
+					URI:         "/datasets/mid-year-pop-est/editions/time-series/versions/1",
+				},
+			})
+		})
+
+		Convey("return an error status if request to GET code-list's editions fails", func() {
```

Comment:
For your tests it's probably worth applying the same changes that Eleanor suggested on my tests:
https://github.com/ONSdigital/dp-frontend-geography-controller/pull/8

It's mostly around the test giving inconsistent description of whether they're just expecting an 'error status' or specifically a '500 status'.